lnwallet/channel.go

```diff
@@ -4955,24 +4932,19 @@ func newOutgoingHtlcResolution(signer input.Signer,
 		Index: uint32(htlc.OutputIndex),
 	}
 
+	// First, we'll re-generate the script used to send the HTLC to
+	// the remote party within their commitment transaction.
+	htlcScriptHash, htlcScript, err := genHtlcScript(
```

Comment:
`genHtlcScript` uses the `isIncoming` and `ourCommit` arguments to determine which scripts to create.  So this commit is really just a reuse of that logic instead of doing the same here.